src/animations.jl

```diff
@@ -6,19 +6,55 @@ struct AnimationTrack{T}
     values::Vector{T}
 end
 
+function Base.insert!(track::AnimationTrack, frame::Integer, value)
+    i = searchsortedfirst(track.frames, frame)
+    insert!(track.frames, i, frame)
```

Comment:
If `i` is already in `track.frames`, then I think we want to replace that value, rather than creating a duplicate.

Reply:
Are you sure? To be clear, this `insert!` method is a straight up rename of the `_setprop!` method that's used in `atframe`,

https://github.com/rdeits/MeshCat.jl/blob/160933c0ae285384da15a9b653010f23046c137d/src/atframe.jl#L1-L5

Reply:
We decided to change both `insert!` and `merge!` to overwrite values set at the same frame.